accessing-data/record-webhooks.mdx

```diff
@@ -129,8 +129,94 @@ Body:
       record_slug: 'ffd928c3-3459-43fb-a2e4-3f1b3ad5fc1f',
       validation_type: 'error',
       column_name: 'email',
-      issue_description: 'Email already exists and must be unqiue.'
+      issue_description: 'Email already exists and must be unique.'
     }]
   }
 }
-```
\ No newline at end of file
+```
+
+## Record Webhook Implementation Example
+
+In this example using Javascript and Express, we create an endpoint in your backend that initially receives the token `http_x_fuse_signature_256` containing the signature generated by Fuse and submitted to you. \
```

Comment:
Remove this entire code description and the code. Just write:

Find sample code below for implementing a record webhook inline and responding with validation issues.

Reply:
![image](https://github.com/flatironsdevelopment/fuse-docs/assets/50498997/f1832d89-3d5c-4393-930a-50544c8dee1b)
Having something like this? @mikefrederick